src/components/util/ErrorBoundary.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+export default class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      hasError: false,
+    };
+  }
+
+  static getDerivedStateFromError(error) {
+    console.log("Error set");
+    return { hasError: true };
+  }
+
+  componentDidCatch(error, info) {
+    console.log(error);
```

Comment:
Should either of these use [`console.error()`](https://developer.mozilla.org/en-US/docs/Web/API/console/error) rather than `console.log()`?